src/applications/simple-forms/20-10207/config/form.js

```diff
@@ -198,25 +203,54 @@ const formConfig = {
           pageClass: 'contact-information',
         },
         mailingAddressPage: {
-          depends: formData => formData.mailingAddressYesNo,
+          depends: formData =>
+            formData.livingSituation.NONE && formData.mailingAddressYesNo,
```

Comment:
maybe want to use a ?.NONE for these

Reply:
Not here; NONE has to be true.  If undefined, this page should not come up.